tests/representation/DataSet/test_kvstore_backed.py

```diff
@@ -80,8 +78,8 @@ def test_add_data(self) -> None:
         mem_kv = MemoryKeyValueStore()
         kvds = KVSDataSet(mem_kv)
 
-        de1 = DataMemoryElement(six.b('bytes1'))
-        de2 = DataMemoryElement(six.b('bytes2'))
+        de1 = DataMemoryElement(bytes('bytes1', 'utf-8'))
```

Comment:
Could this just be `b'bytes1'`? I don't know if I'm missing something where UTF-8 encoding is required or preferred.

Reply:
updated to use b'bytes1'